src/classes/BDI_FieldMappingSet.cls-meta.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+<ApexClass xmlns="urn:metadata.tooling.soap.sforce.com" fqn="BDI_FieldMappingSet">
```

Comment:
What's happening with this switch? I notice several of the meta-xml files get this switch of the attributes but not exactly sure what the significance is. If it's just a shift in format, no big deal - I did a quick google and didn't hit anything that useful on the subject

Reply:
Its due to my sandbox using the new project structure and our repo not being on the new project structure.  I'll need to check on if this automatically gets standardized or whether we need to purposefully use the old syntax.